<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrollableTextDisplay3</class>
 <widget class="QDialog" name="ScrollableTextDisplay3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rank-Order List with reasoning</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="wrapperVerticalLayout">
     <item>
      <widget class="QScrollArea" name="allRoutesScrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="allRoutesscrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>778</width>
          <height>647</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="generateSSHorizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>428</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="allRoutesSelectCheckBox">
         <property name="text">
          <string>select all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="allRoutesGenerateSSPushButton">
         <property name="text">
          <string>generate screenshots</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
